syntax = "proto2";

package StateMachineRawDataProto;

message StateMachineRawData
{
   optional string DownloadFailedSinceTime = 1 [default="0"];
   optional string DownloadState = 2 [default="1"];
   optional string DownloadStateCredit = 3 [default="0"];
   optional string EventStateLastError = 4 [default="0"];
   optional string EventStateLastTime = 5 [default="0"];
   optional string InstallFailedSinceTime = 6 [default="0"];
   optional string InstallState = 7 [default="1"];
   optional string InstallStateCredit = 8 [default="0"];
   optional string LastGoodInstallTime = 9 [default="0"];
   optional bool canSendEvent = 10 [default=true];
}
